tests/unittests/models/conftest.py

```diff
@@ -0,0 +1,331 @@
+from ai21.models import (
```

Comment:
`conftest.py` is usually used for fixtures. Rename file please

Reply:
Done